modules/analytics/src/views/full/index.tsx

```diff
@@ -212,15 +221,15 @@ const Analytics: FC<any> = ({ bp }) => {
   const getReturningUsers = () => {
     const activeUsersCount = getMetricCount('active_users_count')
     const newUsersCount = getMetricCount('new_users_count')
-    const percent = activeUsersCount && (newUsersCount / activeUsersCount) * 100
+    const percent = (activeUsersCount / (newUsersCount + activeUsersCount)) * 100
```

Comment:
can you use Math.round to make sure we don't end up with large number of decimals